<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="4312" started-at="2022-04-27T09:41:43Z" finished-at="2022-04-27T09:41:48Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="4312" started-at="2022-04-27T09:41:43Z" finished-at="2022-04-27T09:41:48Z">
      <class name="ExtentReport.ExtentRep">
        <test-method status="PASS" signature="BT()[pri:0, instance:ExtentReport.ExtentRep@dd3b207]" name="BT" is-config="true" duration-ms="4276" started-at="2022-04-27T15:11:43Z" finished-at="2022-04-27T15:11:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BT -->
        <test-method status="FAIL" signature="testcase1()[pri:0, instance:ExtentReport.ExtentRep@dd3b207]" name="testcase1" duration-ms="9" started-at="2022-04-27T15:11:48Z" finished-at="2022-04-27T15:11:48Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at ExtentReport.ExtentRep.testcase1(ExtentRep.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase1 -->
      </class> <!-- ExtentReport.ExtentRep -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
